rkt/rkt.mk

```diff
@@ -29,15 +29,19 @@ $(BGB_BINARY): $(MK_PATH) | $(BINDIR)
 
 include makelib/build_go_bin.mk
 
+DIST_SRC_FILTER := '_test.go|main.go|_gen.go|_stub.go'
```

Comment:
@krnowak Thanks, I copied from https://github.com/coreos/rkt/pull/2868 @lucab , can probably wait that to be merged first.